scripts/projectsController.js

```diff
@@ -1,8 +1,38 @@
 (function(module) {
   var projectsController = {};
-  projectsController.index = function() {
-    $('.tab-content').hide();
-    $('#projects').show();
+  projectsController.index = function(ctx, next) {
+    projectView.index(ctx.project);
   };
+
+  projectsController.loadProjectSection = function(ctx, next) {
+    var projectData = function(allProjects) {
+      ctx.project = Project.all;
+      next();
+    };
+    if(Project.all.length) {
+      ctx.project = Project.all;
+      next();
+    } else {
+      Project.fetchAll(projectData);
+    }
+  };
+
+  projectsController.loadById = function(ctx, next) {
+
+    if(($('data-id').val()) === ctx.params.id) {
```

Comment:
Your problem will likely start here. I would debug what each of the components of this line are doing, and then see if you can figure what's not matching up. 

You can also trace back and debug the return value of this callback. There may be something not feeding `ctx.project`?